README.md

```diff
@@ -13,8 +13,8 @@ can use [asdf](https://github.com/asdf-vm/asdf) to install it, in case).
 
 You also need a GitHub token, this is used to fetch the list of contributors for
 the repo, you can create one with the following scopes
-`public_repo, read:packages, read:user, repo:status`, by following
-[this guide](https://docs.github.com/en/github/authenticating-to-github/creating-a-personal-access-token).
+`public_repo, read:packages, read:user, repo:status`, by clicking
+[this link](https://github.com/settings/tokens/new?description=Strawberry%20Docs&scopes=repo:status,public_repo,read:user,repo:status,read:packages).
```

Comment:
can we change this to something like: "you can create a token by following this guide or by clicking this link direclty (notice it will ask you for a password)"?

Reply:
done!